cognite/neat/workflows/steps/step_model.py

```diff
@@ -7,6 +8,16 @@
 from cognite.neat.workflows.model import WorkflowConfigItem, WorkflowConfigs
 
 
+class StepCategory(StrEnum):
+    RulesImporter = "rules | importer"
+    RulesParser = "rules | parser"
+    RulesExporter = "rules | exporter"
+    GraphExtractor = "graph | extractor"
+    GraphTransformer = "graph | transformer"
+    GraphStore = "graph | store"
+    GraphLoader = "graph | loader"
```

Comment:
Can we simply use the module name for this grouping instead? That will force all steps to be put into the correct module.

Reply:
removed